postgres/README.md

```diff
@@ -366,7 +366,7 @@ Some of the metrics listed below require additional configuration, see the [samp
 
 See [metadata.csv][20] for a list of metrics provided by this integration.
 
-For Agent version `7.32.0` and later, if you have Database Monitoring enabled, the `postgresql.connections` metric is tagged with `state`, `app`, and `user`.
+For Agent version `7.32.0` and later, if you have Database Monitoring enabled, the `postgresql.connections` metric is tagged with `state`, `app`, `db` and `user`.
```

Comment:
I think we want this known in the `postgresql.connections` metric metadata description as well, so that when someone will see that when reading the docs.

Reply:
OK, i'll add it